beater/interceptors/logging.go

```diff
@@ -58,12 +58,11 @@ func Logging(logger *logp.Logger) grpc.UnaryServerInterceptor {
 			"event.duration", time.Since(start),
 			"grpc.response.status_code", res.Code(),
 		)
-
 		if err != nil {
 			logger.With("error.message", res.Message()).Error(logp.Error(err))
-		} else {
-			logger.Info(res.Message())
+			return nil, err
 		}
-		return resp, err
+		logger.Info("request accepted")
```

Comment:
question: is this the message we want for a successfully handled request?

Reply:
I think it's OK, since the details are already recorded as additional fields. It might be nice to log the number of events accepted, but can just do that as a separate log message in processor/otel.

Reply:
@marclop WDYT?

Reply:
sounds good to me